src/Diagnostics.RuntimeHost/Startup.cs

```diff
@@ -150,18 +150,19 @@ public void ConfigureServices(IServiceCollection services)
                 }).AddJsonOptions(options =>
                 {
                     options.JsonSerializerOptions.WriteIndented = true;
+                    options.JsonSerializerOptions.IncludeFields = true;
```

Comment:
>IncludeFields [](http://example.com/codeflow?start=50&length=13)

very cool change. I was literally changing the fields to properties in other places in the code :D

Reply:
How will this show now in applens output?

Reply:
Without this change, the response from compiler host is empty. Pressing Run never succeeds.

Reply:
Oh so if the compilation output is empty then its considered a failed compilation? Sorry I can't remember